trunner/test_runner.py

```diff
@@ -57,6 +57,7 @@ def run(self):
         for target, tests in self.tests_per_target.items():
             for test_case in tests:
                 test_case.log_test_started()
+                self.runners[target].status('BUSY')
```

Comment:
probably should be in the outer `for`, no need to change state `BUSY -> BUSY` with each new test

Reply:
That's right, thanks

Reply:
not needed at all if setting in constructor